### DISCUSSION TITLE
Should the stackoverflow tag "fancybox" be updated?

### DISCUSSION THREAD

**Discussion by cascading-jox** [fancyapps/ui#121] [General]
In the troubleshooting documentation there is a link to a new tag called "fancyapps-ui" but there is a tag already for "fancybox". Is idea to use this new tag instead of/together with "fancybox" for all things related to the new v4 fancybox? We already have fancybox-2 and fancybox-3, wouldn it make sense to add a fancybox-4 as well? I am not sure what to call fancybox v4 either. As I understand it fancybox is now called fancyapps-ui fancybox and there are two new modules (fancyapps-ui carousel and fancyapps-ui panzoom). I assume "fancyapps-ui" is supposed to be used because it can contain questions about all three modules. 

In my humble opinion I would have wanted "fancyapps-ui" to be called fancybox 4, and the new modules fancybox 4 carousel and fancybox 4 panzoom. For me fancybox is such a strong brand name that it is the synonym for modals/popups and I will probably keep on calling it fancybox.

**Comment by fancyapps** [comment]
Hi,

Thanks for your feedback. It made sense to split Fancybox into 3 separate parts, but naming was a difficult decision. You can still use the "fancybox" tag in Stackoverflow. This project follows the SemVer version guidelines, so the number of versions may increase periodically.
